### ISSUE TITLE
TrackPoint works but Trackpad doesn't

### ISSUE THREAD

**Issue by RyoIkarashi** [tylernguyen/x1c6-hackintosh#31]
Hi, @tylernguyen!
First of all, I really appreciate the great resources.
I'm very new to Hackintosh and managed to get things work well thanks to your instructions.
But the only trackpad function doesn't work at all. I can't event click or move cursor via trackpad and of course gestures don't work either.
I'm pretty glad that you guide me how to get it work if you have time.
What kind of info do you need to figure out the reason it doesn't work?

Here is the current info about my hackintosh.
![Screen Shot 2020-02-04 at 4 47 09](https://user-images.githubusercontent.com/5750408/73685398-73b8f380-4709-11ea-8c59-4881313fc46f.png)

And here are the kexts located in /EFI/Clover/kexts/Other
![Screen Shot 2020-02-04 at 4 54 58](https://user-images.githubusercontent.com/5750408/73686061-a44d5d00-470a-11ea-9595-004291605cc0.png)

**Comment by Porco-Rosso** [comment]
Have you tried pressing the "PrtSc" button next to the right control key? it seems to disable/enable the touch-pad. Would be interested in how to disable this actually...

**Comment by RyoIkarashi** [comment]
@Porco-Rosso thanks for your advice but it didn't work.
Toggling "PrtSc" button disables/enables the TrackPoint functionality but TrackPad (TouchPad) is not disabled/enabled.

**Comment by Porco-Rosso** [comment]
I had some other issues with the trackpad previously, but they went away with a reboot.

In any case I suggest you try the OpenCore EFI method over Clover, as @tylernguyen will no longer be supporting clover in this repo. It is also the general direction the community seems to be heading. Using the EFI files from there might make your trackpad work. 
PS: double-check your BIOS settings, maybe its something set there?

**Comment by RyoIkarashi** [comment]
@Porco-Rosso Thanks for the quick response.
After I switched the bootloder to OpenCore and manually removed kexts caches by `sudo kextcache -i /`, touchpad, trackpad are now working fine including touch gestures even though new issues on Wifi/BT are present this time because I replaced with *Dell DW1560* not *BCM94360CS2* suggested by the author.

Anyway issues on trackpad are now resolved so I close this issue.
Thanks @Porco-Rosso again.
